docs/index.html

```diff
@@ -0,0 +1,72 @@
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Awesome GitHub Copilot - Interactive Explorer</title>
+    <meta name="description" content="Explore and discover GitHub Copilot prompts, instructions, and chat modes">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="icon" href="https://github.githubassets.com/favicons/favicon.svg" type="image/svg+xml">
+</head>
+<body>
+    <header>
+        <a href="#main" class="sr-only">Skip to main content</a>
+        <div class="container">
+            <h1>🤖 Awesome GitHub Copilot</h1>
+            <p class="subtitle">Interactive Explorer for Prompts, Instructions & Chat Modes</p>
+        </div>
+    </header>
+
+    <nav aria-label="Main navigation">
+        <div class="container">
+            <div class="nav-controls">
+                <div class="search-container">
+                    <label for="search" class="sr-only">Search prompts, instructions, and chat modes</label>
+                    <input type="search" id="search" placeholder="Search..." aria-describedby="search-help">
+                    <span id="search-help" class="sr-only">Search across titles and descriptions</span>
+                </div>
+                <div class="filter-container">
+                    <label for="type-filter" class="sr-only">Filter by type</label>
+                    <select id="type-filter" aria-describedby="filter-help">
+                        <option value="all">All Types</option>
+                        <option value="prompts">Prompts</option>
+                        <option value="instructions">Instructions</option>
+                        <option value="chatmodes">Chat Modes</option>
+                    </select>
+                    <span id="filter-help" class="sr-only">Filter content by type</span>
+                </div>
+            </div>
+        </div>
+    </nav>
+
+    <main id="main" class="container">
+        <div class="stats">
+            <span id="total-count">Loading...</span>
+        </div>
+
+        <div class="content-grid">
+            <section id="results" aria-live="polite" aria-label="Search results">
+                <div class="loading">Loading awesome content...</div>
+            </section>
+        </div>
+
+        <div id="pagination" class="pagination" style="display: none;">
+            <button id="prev-page" class="btn btn-secondary" disabled>Previous</button>
+            <span id="page-info"></span>
+            <button id="next-page" class="btn btn-secondary" disabled>Next</button>
+        </div>
+    </main>
+
+    <footer>
+        <div class="container">
+            <p>
+                <a href="https://github.com/github/awesome-copilot">View on GitHub</a> |
+                <a href="https://github.com/github/awesome-copilot/blob/main/CONTRIBUTING.md">Contributing</a> |
+                <a href="https://github.com/github/awesome-copilot/blob/main/LICENSE">License</a>
```

Comment:
can you create links to the repo dynamically depending on the actual repo and user?